rllib/connectors/common/add_states_from_episodes_to_batch.py

```diff
@@ -250,11 +250,20 @@ def __call__(
             # Before adding STATE_IN to the `data`, zero-pad existing data and batch
             # into max_seq_len chunks.
             for column, column_data in data.copy().items():
+                # Do not zero-pad INFOS column.
+                if column == Columns.INFOS:
+                    continue
                 for key, item_list in column_data.items():
-                    if column != Columns.INFOS:
-                        column_data[key] = split_and_zero_pad_list(
-                            item_list, T=self.max_seq_len
-                        )
+                    # Multi-agent case AND RLModule is not stateful -> Do not zero-pad
+                    # for this model.
```

Comment:
bug fix: For multi-agent with some RLModules NOT stateful, we should NOT zero-pad anything.

Reply:
Does this actually work already when using it on full length episodes coming from `OfflineData`?